.stylelintrc.js

```diff
@@ -86,14 +111,21 @@ module.exports = {
             }
         ],
         'font-family-name-quotes': 'always-unless-keyword',
-        // Turn off rules 
+        // Turn off rules
         'selector-class-pattern': null,
+        'selector-id-pattern': null,
         'function-no-unknown': null,
         'property-no-vendor-prefix': null,
         'value-no-vendor-prefix': null,
         'font-family-no-missing-generic-family-keyword': null,
         'scss/dollar-variable-colon-space-after': null,
         'scss/no-global-function-names': null,
-        'value-keyword-case': null
+        'value-keyword-case': null,
+        'media-query-no-invalid': null,
+        'selector-id-pattern': null,
+        'keyframes-name-pattern': null,
+        'scss/dollar-variable-pattern': null,
```

Comment:
A mixture of "changing these could cause problems" and "changing these isn't worth the effort, it doesn't really matter if they match some arbitrary pattern or not"